apis/database/v1alpha1/redshift_types.go

```diff
@@ -0,0 +1,713 @@
+/*
+Copyright 2020 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	runtimev1alpha1 "github.com/crossplane/crossplane-runtime/apis/core/v1alpha1"
+)
+
+// State represents the state of a Redshift cluster.
+type State string
+
+// Redshift cluster states.
+const (
+	// The cluster is healthy and available
+	StateAvailable = "available"
+	// The cluster is being created. The cluster is inaccessible while it is being created.
+	StateCreating = "creating"
+	// The cluster is being deleted.
+	StateDeleting = "deleting"
+	// The cluster is being modified.
+	StateModifying = "modifying"
+	// The cluster has failed and Amazon Redshift can't recover it. Perform a point-in-time restore to the latest restorable time of the Cluster to recover the data.
+	StateFailed = "failed"
+)
+
+// +kubebuilder:object:root=true
+
+// An Redshift is a managed resource that represents an AWS Redshift cluster.
+// +kubebuilder:printcolumn:name="READY",type="string",JSONPath=".status.conditions[?(@.type=='Ready')].status"
+// +kubebuilder:printcolumn:name="SYNCED",type="string",JSONPath=".status.conditions[?(@.type=='Synced')].status"
+// +kubebuilder:printcolumn:name="STATE",type="string",JSONPath=".status.atProvider.clusterStatus"
+// +kubebuilder:printcolumn:name="AGE",type="date",JSONPath=".metadata.creationTimestamp"
+// +kubebuilder:subresource:status
+// +kubebuilder:resource:scope=Cluster,categories={crossplane,managed,aws}
+type Redshift struct {
```

Comment:
I think this resource should actually be `Cluster` and should be in the `redshift.aws.crossplane.io` group

Reply:
Sure. I'll make that change.